src/Equations/elastic/Kernels/Local.cpp

```diff
@@ -52,6 +52,10 @@
 #include <stdint.h>
 #include <cstring>
 
+#include <Kernels/common.hpp>
+GENERATE_HAS_MEMBER(ET)
+GENERATE_HAS_MEMBER(sourceMatrix)
```

Comment:
This macro is also called in Time kernel - maybe we should do this in one place to avoid duplicate code generation (minor issue)